packages/@angular/cli/commands/build.ts

```diff
@@ -31,10 +31,26 @@ export const baseBuildCommandOptions: any = [
     description: 'define the output filename cache-busting hashing mode',
     aliases: ['oh']
   },
+  { name: 'json', type: Boolean, default: false },
 ];
 
 export interface BuildTaskOptions extends BuildOptions {
   watch?: boolean;
+  watcher?: string;
+  supressSizes: boolean;
```

Comment:
Can you remove these changes? They're redundant and should be obtained from `BuildOptions`.